toolkit/smart-contracts/commands/src/d_parameter.rs

```diff
@@ -27,15 +27,15 @@ impl UpsertDParameterCmd {
 		};
 		let client = self.common_arguments.get_ogmios_client().await?;
 
-		upsert_d_param(
+		let result = upsert_d_param(
 			self.genesis_utxo,
 			&d_param,
 			&payment_key,
 			&client,
 			&FixedDelayRetries::two_minutes(),
 		)
 		.await?;
-
+		println!("{}", serde_json::to_value(result)?);
```

Comment:
This will print `null` could we have `{}` if option is none? Both are valid jsons but we use the latter one for permissioned candidates